docs/platform/server-v3/modules/about.md

```diff
@@ -14,100 +14,132 @@ enterprise_only: true
+In previous version of Falcon Server (version 2) there was a distinct separation between extensions, api clients, rest endpoint handlers and event handlers. Each of these needed to be implemented and loaded as a separate package. Falcon Server 3 changes that approach and introduces concept of a module.
+
+So in comparison to Falcon Server 2, Falcon Module it's a container that groups all the above things as one package.
 
 In Falcon Server 2 it was necessary to add multiple packages to have full coverage of features. If we consider shop features that will be handled service X (Magento, BigCommerce, CommerceTools etc) the following packages ere required:
+
 - `@deity/falcon-shop-extension` that provides GraphQL schema for shop features
 - `@deity/falcon-X-api` that provides resolvers and all the calls to service X (implements the shop features)
 - `@deity/falcon-X-endpoints` that provides REST handlers (e.g. webhooks, or callbacks for payments)
 - `@deity/falcon-X-event-handlers` that provides handlers for events in Falcon Server (if such are required)
 
 In Falcon Server you need only:
-- `@deity/falcon-shop-extension` that provides GraphQL schema for shop features
+
+- `@deity/falcon-X-extension` that provides GraphQL schema for shop features
 - `@deity/falcon-X-module` that provides implementation of all the required things in one package
 
 ## Module and extension relation
 
-Please keep in mind that extension packages are still needed, as extension provide GraphQL schema that will be exposed to GraphQL clients. 
+Please keep in mind that extension packages are still needed, as extension provide GraphQL schema that will be exposed to GraphQL clients.
 So now, the 2 things need to be provided:
+
 - extension which is an "interface" for a particular feature
 - module which is an "implementation" of that feature
 
-## Custom modules
+## Configuring module
 
-All the integrations available in Falcon Platform are implemented as extensions + modules. When you want to add new features or change the existing behavior you'll need to add [Extension](../extensions/about) and module that implements features for that Extension.
+In order to enable a module in Falcon Server you need to add its configuration into `config.json` file under `"modules"` section.
 
-Modules can be registered in 2 ways - with auto-discovery mechanism or manually.
+```json
+{
+  "modules": {
+    "my-module": {
+      "package": "<path>"
+    }
+  }
+}
+```
+
+As you an see, Falcon Server expect a key-value map of modules, in most cases order does not matter, but there are exertion, e.g. search modules should be configured last.
+
+We recommend adding it in `config/default.json` file so it will be available no matter what mode (`development`, `production` or any custom) Falcon Server is running. If a module requires some credentials or secrets (sensitive configuration) then that configuration can be added in `config/local.json`. See configuration guide for more details.
```

Comment:
```suggestion
We recommend adding your base configuration in `server/config/default.json` file so it will be available no matter what mode (`development`, `production` or any custom) Falcon Server is running in. If a module requires some credentials that should be secret and not committed to your git repository then that configuration can be added in `server/config/local.json` for local development and as environment variables for remote apps . See configuration guide for more details.
```

Reply:
done